backend/src/main/java/com/javahelp/backend/domain/user/register/RegisterUserResponse.java

```diff
@@ -0,0 +1,70 @@
+package com.javahelp.backend.domain.user.register;
+
+import com.javahelp.model.user.User;
+import com.javahelp.model.user.UserPassword;
+
+public class RegisterUserResponse {
+
+    private User user;
+    private UserPassword password;
+    private String error;
+
+    /**
+     * For bundle up info of the result of this successful registration
+     * @param user the newly created {@link User}
+     * @param password the newly created {@link UserPassword}
+     */
+    public RegisterUserResponse(User user, UserPassword password){
+        this.user = user;
+        this.password = password;
+        this.error = null;
```

Comment:
it's not necessary to set error here, it will be null without this line. If you make it final (which you could), then this would be necessary, but as is it is not needed. That doesn't mean it needs to be fixed, I'm just letting you know for the future.